error-handling/src/main/java/com/palantir/remoting2/errors/SerializableErrorToExceptionConverter.java

```diff
@@ -46,6 +47,7 @@ private SerializableErrorToExceptionConverter() {}
 
     private static final ObjectMapper MAPPER = new ObjectMapper()
             .registerModule(new GuavaModule())
+            .registerModule(new Jdk8Module())
```

Comment:
yep, was stumbling over this one too. will use the extras/jackson-support library. https://github.com/palantir/http-remoting/issues/336